ffmpeg/VITABUILD

```diff
@@ -0,0 +1,54 @@
+pkgname=ffmpeg
+pkgver=3.3.9
+pkgrel=1
+url="http://ffmpeg.org/"
+source=("http://ffmpeg.org/releases/ffmpeg-$pkgver.tar.xz" "ffmpeg.patch")
+sha256sums=('ae34f14fffa65a1a59b256737ca9af7bf4e296b7c4320d42512350126ce06c84' 'c4058a723957547c5450e4c2694eb0b9393d983146079382f311be77f9d87f75')
+
```

Comment:
this package would be required `depends` packages for building.

Reply:
Sorry, what do you mean?

Reply:
imo, this package requires lame. build system couldn't build without depends keyword.